components/form.js

```diff
@@ -1052,5 +1054,43 @@ function Client (Component) {
   }
 }
 
+function PasswordHider ({ onClick, showPass }) {
+  const handleClick = () => {
+    onClick()
+  }
+
+  return (
+    <InputGroup.Text
+      style={{ cursor: 'pointer' }}
+      onClick={handleClick}
+    >
+      {!showPass
+        ? <EyeClose
+            fill='var(--bs-body-color)' height={20} width={20}
+          />
+        : <Eye
+            fill='var(--bs-body-color)' height={20} width={20}
+          />}
+    </InputGroup.Text>
+  )
+}
+
+export function PasswordInput ({ initialValue, label, name, newPass, autoFocus, required }) {
+  const [showPass, setShowPass] = useState(false)
+
+  return (
+    <ClientInput
+      initialValue={initialValue}
+      label={label}
+      name={name}
+      type={showPass ? 'text' : 'password'}
+      autoComplete={newPass ? 'new-password' : 'current-password'}
+      autoFocus={autoFocus}
+      required={required}
+      append={<PasswordHider showPass={showPass} onClick={() => setShowPass(!showPass)} />}
+    />
```

Comment:
I think using `...props` in the function signature and then `{...props}` here  for anything that we simply pass through to `<ClientInput>` would be better